### ISSUE TITLE
AVT- Dropdown list- Focused item is not visible when color is removed

### ISSUE THREAD

**Issue by kcchen-tw** [carbon-design-system/carbon#3121]
## Title 
When colour is removed, the focused item in dropdown list is not visible.

## Environment
> Browser
FF 67.03

## Detailed description
> What did you expect to happen?
- A distinguishable visual focus indicator helps keyboard-only users locate the point of focus.

> What happened instead? 
- The focus is not visible when you try to select any option on the menu.

## Steps to reproduce the issue

1. Step one: To remove color follow  next steps:
For Firefox®:
• Select Tools >Options.
• On the Language and Appearance section select Colors... button.
• On the Colors window go to Override the colors specified by the page with your selections above: and open the menu and select Always, then press OK.
• On the Advance button deselect the Allow pages to choose their own fonts, instead of your selections above checkbox.
• Select OK to save changes.

2. Step two: 
In url: http://react.carbondesignsystem.com/?path=/story/dropdown--default
![3](https://user-images.githubusercontent.com/52026074/59827339-feabd000-936a-11e9-9955-a9304c20b62d.png)

> Please create a reduced test case in CodeSandbox
>
> - Style and vanilla JS:
>   https://codesandbox.io/s/carbon-components-getting-started-t3ec6
> - React:
> https://codesandbox.io/s/codesandbox-bxf8v

## Additional information
It seems only the Vanilla JS has visible focus indicator:
![Screenshot-2019-06-20-14:55:58](https://user-images.githubusercontent.com/52026074/59827559-a0cbb800-936b-11e9-9134-e8d49722e593.png)

- Notes

**Comment by emyarod** [comment]
hi @kcchen-tw thanks for reporting this! yeah it seems the React dropdown changes the background color to indicate the selected item, opened #3124 to add focus outline to highlighted React dropdown items
